modules/troubleshooting-network-observability-loki-resource-exhausted.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-troubleshooting-loki-resource-exhausted_{context}"]
+= Troubleshooting Loki ResourceExhausted error
+Loki may return a "ResourceExhausted" error when network flow data sent by Network Observability exceeds the configured maximum message size. When using the Loki operator, this maximum message size is configured to 100 MiB. 
+
+.Procedure
+. Navigate to *Operators* -> *Installed Operators*, viewing *All projects* from the *Project* dropdown.
+. Look under the *Provided APIs* heading for the Network Observability Operator, select *Flow Collector* then the *YAML view* tab. 
+.. If you are using the Loki Operator, check that the `spec.loki.batchSize` value does not exceed or come close to 100 MiB. You must keep a margin such as 98 MiB max.
+.. If you are using a Loki installation method that is different from the Red Hat Loki Operator, such as Grafana Loki, verify that the `grpc_server_max_recv_msg_size` link:https://grafana.com/docs/loki/latest/configure/#server[Grafana Loki server setting] is higher than the `FlowCollector` `spec.loki.batchSize` configuration. If it is not, you must either increase the server-side limit, or decrease the `FlowCollector` batch size setting to make it lower than the limit. 
```

Comment:
```suggestion
.. If you are using a Loki installation method that is different from the Red Hat Loki Operator, such as Grafana Loki, verify that the `grpc_server_max_recv_msg_size` link:https://grafana.com/docs/loki/latest/configure/#server[Grafana Loki server setting] is higher than the flow collector `spec.loki.batchSize` value. If it is not, you must either increase the `grpc_server_max_recv_msg_size` value, or decrease the `spec.loki.batchSize` value so that it is lower than the limit. 
```
Suggested editing - if possible here I'd use "flow collector" per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references rather than having two items in backticks after one another.

Reply:
Really good suggestion.